profiles/tests/test_views.py

```diff
@@ -48,6 +48,14 @@ def test_edit(self):
         data = {
             "bitbucket_url": "zerg",
             "gitlab_url": "zerg",
+            "extrafields_set-TOTAL_FORMS": 2,
+            "extrafields_set-INITIAL_FORMS": 0,
+            "extrafields_set-MIN_NUM_FORMS": 0,
+            "extrafields_set-MAX_NUM_FORMS": 4,
+            "extrafields-0-key": "Key1",
+            "extrafields-0-value": "Value1",
+            "extrafields-1-key": "Key2",
+            "extrafields-1-value": "Value2",
```

Comment:
:100:

Reply:
I made some updates to the tests for the new field names